frontend/public/components/monitoring/receiver-forms/slack-receiver-form.tsx

```diff
@@ -59,8 +59,8 @@ export const Form: React.FC<FormProps> = ({ globals, formValues, dispatchFormCha
                 'public~Checking this box will write the API URL to the global section of the configuration file where it will become the default API URL for future Slack receivers.',
               )}
             />
-          </div>
-        </div>
+          </GridItem>
+        </Grid>
```

Comment:
It looks like the `Save as default Slack API URL` checkbox is overlapped with `Slack API URL` input